### PR TITLE
Add PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by pt2121**
Hope this would help our code review process

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<!-- Explain how you tested your development -->
+
+- Step 1
+- Step 2
+- Step ...
+
+## Checklist
+- [ ] Changes has been tested on an Android device or Android emulator with API 21
+- [ ] Pull request includes screenshots or videos if the changes impact UI
+- [ ] `connectedCheck` passes
```
Comment (ialokim) [inline .github/PULL_REQUEST_TEMPLATE.md:22]: What do you mean by `connectedCheck`? Maybe some further explanation would be nice.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+## Tests
+
+<!-- Explain how you tested your development -->
+
+- Step 1
+- Step 2
+- Step ...
+
+## Checklist
+- [ ] Changes has been tested on an Android device or Android emulator with API 21
```
Comment (ialokim) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: Maybe also add a checkbox that says the Gradle `automatedTest`s have been checked to succeed.

**Comment by ialokim** [comment]
There is already a PR template for releases in `.github/PULL_REQUEST_TEMPLATE/release.md`. How would they play together? Maybe this one should be moved to the same folder and given a different name?

**Review by ialokim — CHANGES_REQUESTED** [review]

**Comment by pt2121** [comment]
> There is already a PR template for releases in .github/PULL_REQUEST_TEMPLATE/release.md

That one didn't show up for me. I guess I have to pass the query param for the template.
Anyway, I will look into this later when I have time. Closing it now.

**Pull request closed without merging**